politeiawww/totp_test.go

```diff
@@ -0,0 +1,142 @@
+package main
+
+import (
+	"encoding/json"
+	"io/ioutil"
+	"net/http"
+	"net/http/httptest"
+	"testing"
+	"time"
+
+	www "github.com/decred/politeia/politeiawww/api/www/v1"
+	"github.com/pquerna/otp/totp"
+)
+
+func TestSetTOTP(t *testing.T) {
```

Comment:
Unit tests should be testing specific functions where the test function is named `TestFunctionName` and is located in the `_test.go` file that corresponds to the function location.

This is combining a bunch of different things that need to be broken out.  You should have individual test functions for `TestProcessSetTOTP` and `TestProcessVerifyTOTP`.  If you want to test the `handle` functions as well then you should have `TestHandleSetTOTP` and `TestHandleVerifyTOTP` in the `politeiawww_test.go` file.